api/subscriptions/subscriptions.go

```diff
@@ -49,6 +52,11 @@ const (
 )
 
 func New(repo *chain.Repository, allowedOrigins []string, backtraceLimit uint32, txpool *txpool.TxPool) *Subscriptions {
+	cacheSize := backtraceLimit * 120 / 100
```

Comment:
This may be problematic if node hosters set a very large number for the backtrace limit.

The idea around the `120` is to add %20 extra capacity to the backtrace limit.

Should we set a constant max instead of using the backtrace?